README.md

```diff
@@ -77,7 +77,7 @@ Learn more in [Getting Started](https://google.github.io/mesop/getting-started/i
 
 ## Reporting a security issue
 
-If you identify a security vulnerability with Mesop, please file a [GitHub security issue](https://github.com/google/mesop/security/policy) and we will promptly respond to it.
+If you identify a security vulnerability with Mesop, please file a [GitHub security issue](https://github.com/google/mesop/security/advisories/new) and we will promptly respond to it.
```

Comment:
For some reason when I go to that link I get a 404.

Reply:
Hm... interesting I see this page: 
<img width="1413" alt="image" src="https://github.com/user-attachments/assets/f66ee986-bc9c-4da4-828e-51c247481ec3" />

I checked out repo settings and it looks like Google has enforced disabling Private Vulnerability Reporting: https://github.com/google/mesop/settings/security_analysis

I'm going to close this PR - it's kind of unfortunate because we effectively have no way of getting security reports for this project.